lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -63,7 +65,7 @@
     description:
       - The port number on which each of the cache nodes will accept connections
     required: false
-    default: None
+    default: 11211
```

Comment:
Now I changed the default value to None again.
It is checking for None before the boto call and if it is, setting it to the default port for redis or memcached depending on which is set.